internal/rpmmd/repository.go

```diff
@@ -150,6 +156,39 @@ func LoadRepositories(confPaths []string, distro string) (map[string][]RepoConfi
 		return nil, err
 	}
 
+	if strings.HasPrefix(distro, "rhel-8") {
+		var sslClientKey string
+		var sslClientCert string
+
+		regexKey := regexp.MustCompile("^[0-9]*-key.pem")
+		regexCert := regexp.MustCompile("^[0-9]*.pem")
```

Comment:
Good idea to restrict which files we're reading! `MustCompile` panics when the passed regexp is invalid. It's probably better to move this into a global variable, so that that panic happens when starting the program.

Also ,`[0-9]+` :wink:

Reply:
The regex is global and fixed now.